packages/fxa-settings/src/components/DropDownAvatarMenu/index.tsx

```diff
@@ -81,7 +83,9 @@ export const DropDownAvatarMenu = () => {
                   <Avatar className="w-10" />
                 </div>
                 <p className="leading-5 max-w-full truncate">
-                  <span className="text-grey-400 text-xs">Signed in as</span>
+                  <Localized id="drop-down-menu-signed-in-as">
```

Comment:
IIRC the right way to update this section to handle the $user variable in the string is to insert the `<span>displayName</span>` using the `vars` attribute:

```
<Localized id="drop-down-menu-signed-in-as" 
  vars={{
    user: ( <span className="font-bold block truncate"
                    data-testid="drop-down-name-or-email"
                  >
                    {displayName || primaryEmail.email}
                </span>
     )
  }}
>
  <span className="text-grey-400 text-xs">Signed in as { $user }</span>
</Localized>
```

I might be mixing up `elems` and `vars` here, but I think this is roughly how it should work. See the Localized docs for details: https://github.com/projectfluent/fluent.js/wiki/Localized